gnupg/_parsers.py

```diff
@@ -989,10 +988,12 @@ def key(self, args):
             self.curkey[vars[i]] = args[i]
         self.curkey['uids'] = []
         self.curkey['sigs'] = {}
+        self.curkey['rev'] = {}
         if self.curkey['uid']:
             self.curuid = self.curkey['uid']
             self.curkey['uids'].append(self.curuid)
             self.sigs[self.curuid] = set()
+            self.rev[self.curuid] = set()
```

Comment:
Where is this self.rev created? It might need to be created in the `__init__`.

Isn't it having a name collision with the `rev` method defined below? Maybe this dict could be renamed to `revs`?